server.js

```diff
@@ -34,6 +34,8 @@ const EmailUtils = require('./email-utils')
 const HBSHelpers = require('./template-helpers/')
 const HIBP = require('./hibp')
 const IpLocationService = require('./ip-location-service')
+const { initMonthlyCron } = require('./cron')
+const { sendUnresolvedBreachEmails } = require('./scripts/send-email-to-unresolved-breach-subscribers')
```

Comment:
change (blocking): Oh wow, so this will run every month and it will run in the app server process?! That means it will slow the Monitor site down while it's sending tons of emails.

We should work with SRE to move this to a "real" cron job that is de-coupled from the app server process.

Reply:
Changing this to non-blocking so long as give ourselves fast-follow work to move this to a cron job with SRE support.